jsonnet/rules.libsonnet

```
@@ -609,6 +609,10 @@ function(params) {
           expr: 'max by(scope)(vsphere_infrastructure_failure_domains)',
           record: 'cluster:vsphere_infrastructure_failure_domains:max',
         },
+        {
+          expr: 'max by(status)(vsphere_csi_migration)',
```

```suggestion
          expr: 'max by(status)vsphere_csi_migration{status=~"|LegacyDeprecatedInTreeDriver|CSIWithMigrationDriver"})',
```
Since we already know the possible label value set, lets be explicit here. I understand this is also constrained on the openshift/api level, but we'd like to establish this sort of aggressive cardinality specification for telemetry rules.

---

Fixed